tests/README.md

```diff
@@ -14,6 +14,11 @@ $ docker compose -f docker-compose.e2e.yml up
 $ npm run test:e2e
 ```
 
+### Occasional Failures
+
+- Automated end-to-end tests require the `SESSION_TOKEN`, and may fail unexpectedly, due to rotating session tokens. To resolve this, log into local/dev/staging. Locate the session token in dev tools: Application -> cookies -> \_\_Secure-next-auth.session-token -> value.
```

Comment:
What is meant by "log into local/dev/staging"?

Reply:
Ah, so the tests are relying on a valid token from an individual's login, which seem to have an expiry date of one month from creation.

I would expect to encounter this problem about once a month then 🤔